Neuter all kvm(3) calls until a proper fix is found; this allows
gnome-system-monitor to report net/disk usage again instead of simply
dumping core (abort() for callers).

Index: sysdeps/openbsd/ppp.c
--- sysdeps/openbsd/ppp.c.orig
+++ sysdeps/openbsd/ppp.c
@@ -68,6 +68,9 @@ static struct nlist nlst [] = {
 void
 _glibtop_init_ppp_p (glibtop *server)
 {
+	glibtop_warn_io_r (server, "kvm_nlist (i4bisppp_softc)");
+	return;
+
 #ifdef HAVE_I4B
 #ifdef HAVE_I4B_ACCT
 	server->sysdeps.ppp = _glibtop_sysdeps_ppp |
@@ -97,6 +100,8 @@ glibtop_get_ppp_p (glibtop *server, glibtop_ppp *buf, 
 	glibtop_init_p (server, (1L << GLIBTOP_SYSDEPS_PPP), 0);
 
 	memset (buf, 0, sizeof (glibtop_ppp));
+
+	return;
 
 	if (kvm_read (server->machine->kd, nlst [0].n_value,
 		      &data, sizeof (data)) != sizeof (data))
